src/components/ContactForm/ContactForm.hooks.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState, useContext } from 'react'
+import { FormDataType, SnackbarContext } from '@services'
+import { useTranslation } from 'react-i18next'
+
+export type useContactFormHookParams = {
+  defaultReportType?: string
+  defaultReportTopic?: string
+  defaultDescription?: string
+}
+
+export type ContactFormHookReturn = {
+  readonly reportType: string
+  readonly reportTopic: string
+  readonly description: string
+  readonly setReportType: (reportType: string) => void
+  readonly setReportTopic: (reportTopic: string) => void
+  readonly setDescription: (description: string) => void
+  readonly submit: (content: FormDataType) => void
+}
+/**
+ * Hook for the ContactForm logic. Handles reporttype, reporttopic and description state which sets the input for the textfields and
+ * provides function to submit the form.
+ * @param params - The default values for the form.
+ * @returns {ContactFormHookReturn} - The form logic.
+ * @function submit - Function for submitting the form. It writes the values of the form into the responseBody.
+ * The responseBody is the object that will be sent to the backend.
+ */
+export const useContactForm = (params?: useContactFormHookParams): ContactFormHookReturn => {
+  const { t } = useTranslation()
+  const { addSnackbar } = useContext(SnackbarContext)
+
+  // ** State **//
+  const reportTypes = t('components.ContactForm.types', {
+    returnObjects: true
+  }) as [{ value: string; label: string }]
+  const {
+    defaultReportType = '', //TODO: TEST reportTypes.at(-1)?.value ??
+    defaultReportTopic = '',
+    defaultDescription = ''
+  } = params || {}
```

Comment:
better wegen etwaiger bugs
```suggestion
  } = params ?? {}
```